Content/install/liquibase-macos.htm

```diff
@@ -24,14 +24,14 @@ <h3>Automatic</h3>
                         <p>If the <MadCap:variable name="General.Liquibase" /> Installer does not start automatically after downloading, locate the Installer file in your <code>Downloads</code> directory and run it.</p>
                         <h3>Manual</h3>
                         <ol>
-                            <li>Click <a href="https://github.com/liquibase/liquibase/releases/download/v4.10.0/liquibase-4.10.0.tar.gz">Download <MadCap:variable name="General.Liquibase" style="font-weight: normal;" /></a> to download the <MadCap:variable name="General.Liquibase" /> setup files.</li>
+                            <li>Click <a href="https://github.com/liquibase/liquibase/releases/download/v4.10.0/liquibase-4.10.0.tar.gz">Download <MadCap:variable name="General.Liquibase" style="font-weight: normal;" /></a> to download the <MadCap:variable name="General.Liquibase" /> CLI files.</li>
```

Comment:
Will this version link automatically update to latest for every release?

Reply:
I assume it must? Or is known to be updated by the doc team as part of the release? I just updated what was there and didn't touch the link....

Reply:
I assumed as much as well. @cmmay or @adrian-velonis1, are we making a correct assumption about the updating of that link?